src/components/singlEventPageComponents/EditEvent.jsx

```diff
@@ -0,0 +1,164 @@
+import { doc, updateDoc } from "firebase/firestore";
+import { useRouter } from "next/router";
+import React from "react";
+
+import { firestore } from "@/lib/firebase/controller";
+
+function EditEvent() {
+    const router = useRouter();
+    const { id } = router.query;
+
+    function handleEditForm(e) {
+        e.preventDefault();
+        // console.log(e.target.about.value)
+        // console.log(e.target.date.value)
+        // console.log(e.target.image.value)
+        // console.log(e.target.category.value)
+        // console.log(e.target.location.value)
+        if (id) {
```

Comment:
you can change to id? if it's causing reliad issues, had a similar a problem while working on the edit profile, i believe last time i saw you having this issue with the id not being readble for having a value of null (i can't see the preview since vercel deployment failed)

Reply:
i also believe you don't need to condition the handleSubmit button, since the edit event page should only be accessible by the creator himself, it's the edit button that should be conditioned according to it (need pair revision)

Reply:
i think you are right , we can look to into it

Reply:
actually it's to make sure the id fully exists therefore not making any errors